### PR TITLE
Fix sentence negation in Step 2

### PR DESCRIPTION
**Pull request by navsgh**

### CODE REVIEW

**Comment by tesseralis** [comment]
No, the original version is correct. It's saying "(Even if) you're familiar with state, don't use it for this project". I agree it might cause some confusion. What do you think we should reword it to? Should we just delete that clause?

**Comment by navsgh** [comment]
If that's the case, I think we can delete the sentence "If you’re familiar with the concept of state, don’t use state at all to build this static version".  Also, the last sentence "Since this is a static version of the app, you don’t need it." clarifies it. 

If we want to keep it we can change it to "Don’t use state at all to build this static version".

**Comment by navsgh** [comment]
@tesseralis Please review the new commit. If that's better can you please close this? Thanks.

**Comment by alexkrolick** [comment]
It's trying to say "don't use state to build this part (and if this is the first time you've heard of 'state' don't worry about it, we'll get to it".

Can you come up with something that preserves that meaning?

**Comment by navsgh** [comment]
Updated the description. Please review
